release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -419,6 +419,12 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4.18-installation-and-update-ovn-kubernetes-join-subnet_{context}"]
+==== Install-time configuration for the `ovn-kubernetes` join subnet feature is enabled
```

Comment:
```suggestion
==== Configuring the ovn-kubernetes join subnet during cluster installation
```

OR

```suggestion
==== Configuring the IPV4 join subnet during cluster installation
```

Instead of "Install-time" I think "during cluster installation" is clearer.